### PR TITLE
Add documentation for three macros

### PR DESCRIPTION
**Pull request by woutervdub**
crossbox, pleat, and sewTogether

### CODE REVIEW

**Review by SeaZeeZee — APPROVED** [review]
This looks great to me!

File: markdown/dev/reference/macros/pleat/en.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  return part
+}
+```
+</Example>
+
+## Configuration
+
+| Property        | Default  | Type                | Description |
+|----------------:|----------|---------------------|-------------|
+| `from`          |          | [Point](/reference/api/point) | The top left point of the crossbox |
```
Comment (BenJamesBen) [inline markdown/dev/reference/macros/pleat/en.md:54]:
```suggestion
| `from`          |          | [Point](/reference/api/point) | The start point of the pleat |
```

File: markdown/dev/reference/macros/pleat/en.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+| `to`            |          | [Point](/reference/api/point) | The bottom right point of the crossbox |
```
Comment (BenJamesBen) [inline markdown/dev/reference/macros/pleat/en.md:55]:
```suggestion
| `to`            |          | [Point](/reference/api/point) | The end point of the pleat |
```

File: markdown/dev/reference/macros/sewtogether/en.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
+---
+title: sewtogether
```
Comment (BenJamesBen) [inline markdown/dev/reference/macros/sewtogether/en.md:2]:
```suggestion
title: sewTogether
```

File: markdown/dev/reference/macros/sewtogether/en.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+---
+
+The `sewtogether` macro is used to mark where two parts of the same `part` need
```
Comment (BenJamesBen) [inline markdown/dev/reference/macros/sewtogether/en.md:5]:
```suggestion
The `sewTogether` macro is used to mark where two parts of the same `part` need 
```

File: markdown/dev/reference/macros/sewtogether/en.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+</Example>
+
+## Configuration
+
+| Property        | Default  | Type                | Description |
+|----------------:|----------|---------------------|-------------|
+| `from`          |          | [Point](/reference/api/point) | One side of what needs to be sewn together |
+| `to`            |          | [Point](/reference/api/point) | The other side of what needs to be sewn together |
+| `middle`        | null     | [Point](/reference/api/point) | The middle point (when ommitted, it will be halfway between `from` and `to`) |
+| `prefix`        | 'pleat'  | String   | The prefix to be used for creating all the points and paths |
```
Comment (BenJamesBen) [inline markdown/dev/reference/macros/sewtogether/en.md:60]:
```suggestion
| `prefix`        | 'sewtogether'  | String   | The prefix to be used for creating all the points and paths |
```

**Review by BenJamesBen — CHANGES_REQUESTED** [review]
You'll probably want to fix all the typos and small corrections anyway, but the only change I actually am requesting is to correct the instances of "sewtogether" to "sewTogether". The capitalization difference actually caused an issue during my testing, where it took me a while to figure out why the "sewtogether" macro wasn't working for me.

File: markdown/dev/reference/macros/crossbox/en.md [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+|-------------------|-------------|
+| `points.${id}_boxTopLeft` | Point Top Left of the outer box |
+| `points.${id}_boxTopRight` | Point Top Right of the outer box |
+| `points.${id}_boxBottomLeft` | Point Bottom Left of the outer box |
+| `points.${id}_boxBottomRight` | Point Bottom Right of the outer box |
+| `points.${id}_topCrossTL` | Point Top Left of the inner box |
+| `points.${id}_topCrossTR` | Point Top Right of the inner box |
+| `points.${id}_topCrossBL` | Point Bottom Left of the inner box |
+| `points.${id}_topCrossBR` | Point Bottom Right of the inner box |
+| `points.${id}_textAnchor` | Point for the text |
```
[the remark this answers is not in the document]
Comment (woutervdub) [reply markdown/dev/reference/macros/crossbox/en.md:73]: This has been fixed this morning.

[the remark this answers is not in the document]
Comment (woutervdub) [reply markdown/dev/reference/macros/crossbox/en.md:73]: (in PR #3999)

**Review by BenJamesBen — APPROVED** [review]

**Comment by SeaZeeZee** [comment]
@joostdecock This can also be merged!

**Pull request merged**